packages/core/src/wallet/metamaskBridge.ts

```diff
@@ -7,6 +7,13 @@ import type {
 import wallets, { WalletProvider } from "../discovery"
 import { init, loadRemote } from "@module-federation/runtime"
 
+const remoteEntryUrl =
+  import.meta.env.MODE === "development"
+    ? "http://localhost:8082/remoteEntry.js"
+    : "https://snaps.consensys.io/starknet/get-starknet/v1/remoteEntry.js"
+
+console.log(remoteEntryUrl)
```

Comment:
shall we remove the console.log?

beside, do you think this is better to using a ENV var , rather than hardcode url? hence we dont need to update the cmd of other package

something like 
```
const remoteEntryUrl = process.env.MM_FED_URL ?? "https://snaps.consensys.io/starknet/get-starknet/v1/remoteEntry.js"
```